src/utilities/__tests__/extendSchema-test.js

```diff
@@ -314,24 +372,54 @@ describe('extendSchema', () => {
       extend type Foo {
         deprecatedField: String @deprecated(reason: "not used anymore")
       }
+
+      extend enum SomeEnum {
+        DEPRECATED @deprecated(reason: "do not use")
+      }
```

Comment:
Test name is "extends objects with deprecated fields"
So I think it's better to create a separate one for enum values.